app/main.py

```diff
@@ -2,0 +1,14 @@
+
+def copy_file(command: str) -> None:
+    command_parts = command.split()
+    if command_parts[0] != "cp":
+        print("Invalid command format.")
+        return
+    source_file, destination_file = command_parts[1], command_parts[2]
+    if source_file == destination_file:
+        print("Source and destination files are the same.")
+        return
+    else:
+        with (open(source_file, "r") as file_in,
```

Comment:
Code Style: It's not common to use parentheses around the `with` statement in Python. You can remove the parentheses for better readability.